src/Console/Command/CommandHelp.php

```diff
@@ -420,7 +430,8 @@ function (FixerOptionInterface $optionA, FixerOptionInterface $optionB) {
             }
         }
 
-        return $help;
+        // prevent "\</foo>" from being rendered as an escaped literal style tag
+        return preg_replace('#\\\\(</.*?>)#', '<<$1', $help);
```

Comment:
Can you create a test scenario for this changes? I cannot create a scenario where when removing this the output changes.